transferui/src/androidTest/java/com/hyperwallet/android/ui/transfer/rule/rule/HyperwalletExternalResourceManager.java

```diff
@@ -0,0 +1,73 @@
+package com.hyperwallet.android.ui.transfer.rule.rule;
+
+import org.junit.rules.TestWatcher;
+import org.junit.runner.Description;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.net.URL;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class HyperwalletExternalResourceManager extends TestWatcher {
```

Comment:
I thought we'd move this class and the others below to the testutils project...

Reply:
The latest development wasn't merged to the epic yet.  Was going to fix it together in one PR.

Reply:
ok